### PR TITLE
Refactor part 2

### PR DESCRIPTION
**Pull request by Devoxin**
• Kumo
• More docs
• Cleaned up/fixed some more code
• Minor changes to setup.py and how the library is installed
• Travis stuff
• Cleaned up readme

### CODE REVIEW

**Comment by TAG-Epic** [comment]
Bump ig

File: README.md [hunk 1]
```diff
@@ -1,19 +1,25 @@
-# KSoft.Si API
+<img align="right" src="https://cdn.ksoft.si/images/Logo1024-W.png" height="70" width="70">
```
Comment (AndyTempel) [inline README.md:1]: pls change src to this: https://cdn.ksoft.si/images/ksoft-logo-text.png

**Comment by AndyTempel** [comment]
imagine removing install requirements from travis
https://i.imgur.com/HfFiXUL.png

**Comment by Devoxin** [comment]
shush
i fixed it anyway

**Pull request merged**